packages/@o3r/eslint-plugin/package.json

```diff
@@ -31,6 +31,9 @@
     "type-fest": "^4.10.2"
   },
   "peerDependenciesMeta": {
+    "json-diff": {
```

Comment:
not really optional in the current implementation: you should turn its import to dynImport or not export diff-import in public_apii